documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -58,246 +86,90 @@ base-char = 1-9 A-H J-N P-Z a-k m-z
-
-The IOTA Identity implementation currently supports:
-- `JcsEd25519Signature2020` 
-
-It must be emphasized that the IOTA Identity implementation of  `JcsEd25519Signature2020` follows the [official specification](https://identity.foundation/JcsEd25519Signature2020/) with the single exception that the SHA-256 pre-hash of the input, the third step of the [proof generation algorithm](https://identity.foundation/JcsEd25519Signature2020/#ProofGeneration), is skipped. See the following two GitHub issues [#22](https://github.com/decentralized-identity/JcsEd25519Signature2020/issues/22) and [#26](https://github.com/decentralized-identity/JcsEd25519Signature2020/issues/26) for more context on why this deviation is deemed necessary. 
-
-**Verification Method**
-
-The proof object MUST include a `verificationMethod` which references a verification method embedded in the same DID Document. The verification method must be capable of signing DID messages as per the [Signing Keys](#signing-keys) section.
-
-**Signature**
-
-Depending on the verification method, a set of fields are REQUIRED to provide the cryptographic proof of the DID Document. For example, the `JcsEd25519Signature2020` method has a `signatureValue` field. 
-
-Example `proof` using the `JcsEd25519Signature2020` method:
-```json
-"proof": {
-    "type": "JcsEd25519Signature2020",
-    "verificationMethod": "did:iota:GzXeqBXGCbuebiFtKo4JDNo6CmYmGbqxyh2fDVKadiBG#authentication",
-    "signatureValue": "3fLtv3KUU4T5bHNLprV3UQ2Te3bcRZ9uUYSFouEA7fmYthieV35NNLqbKUu8t2QmzYgnfp1KMzCqPzGNi3RjU822"
-}
 ```
 
-### Anatomy of the Diff object
+## Controllers
 
-The `diff` object MUST contain all the differences between the current and previous DID Document and its DID Document Metadata. The differentiation is formatted as a JSON object that includes the differences between the two DID Document objects and the two DID Document Metadata objects. Exact details of how this is generated will be added later.
+As of now, only one state and one governance controller can be set for an Alias Output. Multiple controller support can be done by future updates.
 
-Example `diff` of adding a new service entry to the document and changing the `updated` field in the metadata:
-```json
-{
-  "diff": {
-    "doc": {
-      "service": [
-        {
-          "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e#linked-domain-1",
-          "type_": "LinkedDomains",
-          "service_endpoint": "https://example.com/"
-        }
-      ]
-    },
-    "meta": {
-      "updated": "2022-01-21T09:50:28Z"
-    }
-  }
-}
-```
+A state controller can directly update the DID Document and the amount of IOTA coins held by the Output, but it can't destroy the output. A governor controller, one the other hand, can indirectly update the DID Document by updating the state controller. It also can destroy the output by having a governance transition to the empty state.
```

Comment:
Place this above the previous paragraph. Avoid saying "empty state" for destruction, it can cause confusion because that is how we deactivate DID Documents now, by publishing an empty `stateMetadata`.

```suggestion
A state controller can directly update the DID Document and the amount of coins held by the Alias Output, but it cannot destroy the output. A governor controller, one the other hand, can indirectly update the DID Document by updating the state controller. It can also destroy the output by performing a governance transition to the empty state.
```